src/style/mixins.scss

```diff
@@ -340,3 +340,24 @@
         }
     }
 }
+
+/**
+* This mixin creates a cross-browser font stack.
+* - `sans-serif` can be used for the UI of the components.
+* - `monospace` can be used for code.
+*
+* ⚠️ If we change the font stacks, we need to update
+* 1. the consumer documentation in `README.md`, and
+* 2. the CSS variables of `--kompendium-example-font-family`
+* in the `<style>` tag of `index.html`.
+*/
+@mixin font-family($letterform) {
+    @if $letterform == sans-serif {
+        font-family: ui-sans-serif, system-ui, sans-serif;
+    } @else if $letterform == monospace {
+        font-family: ui-monospace, 'Cascadia Code', 'Source Code Pro', Menlo,
+            Consolas, 'DejaVu Sans Mono', monospace;
+    } @else {
+        @error "Please specify the font-family #{$style}!";
```

Comment:
What does `#{$style}` refer to? Should it say `#{$letterform}` instead?

Reply:
🙈 copy/paste mistake

Yes, it should say `#{$letterform}`